File: src/main/java/um/haberes/core/model/extern/EjercicioDto.java
package um.haberes.core.model.extern;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import java.time.OffsetDateTime;

@Getter
@Setter
@EqualsAndHashCode
@ToString
@NoArgsConstructor
@AllArgsConstructor
public class EjercicioDto {

    private Integer ejercicioId = null;

    private String nombre = null;

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssXX", timezone = "UTC")
    private OffsetDateTime fechaInicio = null;

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssXX", timezone = "UTC")
    private OffsetDateTime fechaFinal = null;

    private Byte bloqueado = 0;

    private Integer ordenContableResultado = null;

    private Integer ordenContableBienesUso = null;
}

File: src/main/java/um/haberes/core/kotlin/model/extern/EjercicioDto.kt
package um.haberes.core.kotlin.model.extern

import com.fasterxml.jackson.annotation.JsonFormat
import java.time.OffsetDateTime

data class EjercicioDto(

    var ejercicioId: Int? = null,
    var nombre: String? = null,
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssZ", timezone = "UTC")
    var fechaInicio: OffsetDateTime? = null,
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssZ", timezone = "UTC")
    var fechaFinal: OffsetDateTime? = null,
    var bloqueado: Byte = 0,
    var ordenContableResultado: Int? = null,
    var ordenContableBienesUso: Int? = null

)
